src/Model/Todo/TodoReminderStatus.php

```diff
@@ -22,6 +22,6 @@
  */
 final class TodoReminderStatus extends Enum
 {
-    const OPEN = "open";
-    const CLOSED = "closed";
+    const OPEN = "OPEN";
```

Comment:
@BartoszBartniczak With local installations I mean the ones of our users. Many people have installed prooph-do on their dev machines and if we change the values of the constants, the installation may break, because these values are written to the database and cannot be read again, when we change them.
prooph-do is only a learning app so not a big issue, but I'd like to find out why this change is needed before I merge it. Something has changed between the last master build and now. If we know what has changed and we know that changing the constant values is the only way to go, then we can do it. Unfortunately, I'm busy at the moment. Give me one or two more days to figure that out. Thank you for your work so far.